pageserver/src/repository.rs

```diff
@@ -1,177 +1,9 @@
 use crate::walrecord::NeonWalRecord;
-use anyhow::{bail, Result};
-use byteorder::{ByteOrder, BE};
 use bytes::Bytes;
 use serde::{Deserialize, Serialize};
-use std::fmt;
-use std::ops::{AddAssign, Range};
+use std::ops::AddAssign;
 use std::time::Duration;
 
-#[derive(Debug, Clone, Copy, Hash, PartialEq, Eq, Ord, PartialOrd, Serialize, Deserialize)]
```

Comment:
Should this be moved back or not?

Reply:
I cannot, if I continue with the "file names in `models` module" approach: `DeltaLayer` has a `Range<Key>` inside of it, so `Key` has to be in the `models` module too.

Reply:
Yeah I wasn't expecting the LayerFileName to move either, but since it has now been done... Like I commented, probably doesn't matter so much where these are in the monorepo after all. Sadly the comments describing key are now a more far away but that's a problematic structure anyway. (Checked that those are referenced as `pgdatadir_mapping.rs` which is still accurate.)

At least we are not serializing Key's anymore which I think is more important -- I don't see value in that breakdown into fields.

Alternatives would had included:
- be generic over `LayerName: Serialize`
- `String`

Reply:
Now it's a `String`.